src/seccomp/policy/DefaultPolicy.cc

```diff
@@ -44,7 +44,9 @@ void DefaultPolicy::addExecutionControlRules(bool allowFork) {
              "clock_nanosleep",
              "open",
              "epoll_create1",
-             "openat"});
+             "openat",
+             "newfstatat",
+             "pread64"});
```

Comment:
`pread64` should go in `addInputOutputRules()`
`newfstatat` should be in `addFileSystemAccessRules()`